hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/WritableStoreFactory.java

```diff
@@ -63,13 +64,24 @@ public WritableTopicStore createTopicStore() {
      */
     @NonNull
     public WritableTokenStore createTokenStore() {
-        final var tokenStates = stateAccessor.getHederaState().createWritableStates(TokenService.NAME);
+        final var tokenStates = stateAccessor.getHederaState().createWritableStates(TokenServiceImpl.TOKENS_KEY);
         return new WritableTokenStore(tokenStates);
     }
 
     @NonNull
     public WritableTokenRelationStore createTokenRelStore() {
-        final var tokenRelStates = stateAccessor.getHederaState().createWritableStates("TokenRelations");
+        final var tokenRelStates = stateAccessor.getHederaState().createWritableStates(TokenServiceImpl.TOKEN_RELS_KEY);
```

Comment:
I think the names should be defined in the interface.

Reply:
Changed them back to TokenService.NAME